test/ApplyReleasesTests.cs

```diff
@@ -276,6 +276,10 @@ public async Task ApplyReleaseWhichRemovesAFile()
                 string packagesDir = Path.Combine(appDir, "packages");
                 Directory.CreateDirectory(packagesDir);
 
+                Utility.ExtractZipToDirectory(
+                    IntegrationTestHelper.GetPath("fixtures", "Squirrel.Core.1.1.0.0-full.nupkg"),
```

Comment:
What are those hard-coded version numbers?

Reply:
The Squirrel has a special "fixtures" directory which is shared place for the unit tests nupkgs. There are prepared nupkgs of various versions. This is how to use one of them.